airflow/utils/strings.py

```diff
@@ -27,4 +27,4 @@ def get_random_string(length=8, choices=string.ascii_letters + string.digits):
 
 def to_boolean(astring):
```

Comment:
Could you add a type annotation here? With the additional possibility of `None` this is less straightforward than before.